### PR TITLE
[Core > operations] Add Laravel for operations and improve doc tips

### PR DESCRIPTION
**Pull request by vinceAmstoutz**
Add Laravel for operations and improve doc tips

### CODE REVIEW

**Comment by vinceAmstoutz** [comment]
@dunglas I had to put `---` between 2 block quotes so that the IC wouldn't explode. For more info [Markdown lint doc about blank line inside blockquote](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md028---blank-line-inside-blockquote).

**Comment by vinceAmstoutz** [comment]
> @dunglas I had to put `---` between 2 block quotes so that the CI wouldn't explode. For more info [Markdown lint doc about blank line inside blockquote](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md028---blank-line-inside-blockquote).

File: core/operations.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ an implementation point of view, an operation is a link between a resource, a ro
 <p align="center" class="symfonycasts"><a href="https://symfonycasts.com/screencast/api-platform/operations?cid=apip"><img src="../symfony/images/symfonycasts-player.png" alt="Operations screencast"><br>Watch the Operations screencast</a></p>
 
 API Platform automatically registers typical [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations
-and describes them in the exposed documentation (Hydra and Swagger). It also creates and registers routes corresponding
-to these operations in the Symfony routing system (if it is available).
+and describes them in the exposed documentation (Hydra and Swagger). It also creates and registers routes
```
Comment (dunglas) [inline core/operations.md:9]:
```suggestion
and describes them in the exposed documentation (Hydra and OpenAPI). It also creates and registers routes
```

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**